components/mysql/actions/execute-query/execute-query.mjs

```diff
@@ -0,0 +1,58 @@
+import commonTable from "../common/table.mjs";
+
+const { mysql } = commonTable.props;
+
+export default {
+  ...commonTable,
+  key: "mysql-execute-query",
+  name: "Execute Query",
+  description: "Find row(s) via a custom query. [See the docs here](https://dev.mysql.com/doc/refman/8.0/en/select.html)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    ...commonTable.props,
```

Comment:
Since you are importing the prop for `app`. I suggest you change the property description here. The actual description is **The database table to watch for changes**. I guess something more specific would be great. Something like: **Table to execute query** or something like this